src/recipe/Baked.Recipe.Service.Application/Authentication/FixedBearerToken/FixedBearerTokenAuthenticationFeature.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Baked;
 using Baked.Architecture;
+using Baked.Runtime.Configuration;
 using Microsoft.AspNetCore.Authentication;
```

Comment:
why has this extra namespace become necessary? probably there is a wrong namespace instead of `Baked` somewhere in extensions... :eyes:

Reply:
changed file namespaces to `Baked.Runtime`